src/AddTodoForm.js

```diff
@@ -1,10 +1,21 @@
 import React from 'react';
 
-const AddTodoForm = () => {
+const AddTodoForm = (props) => {
+  const handleAddTodo = (event) => {
+    event.preventDefault();
+    const form = event.target;
+    const title = document.getElementById('todoTitle');
```

Comment:
Clever! This will do the trick but in React you want to avoid directly accessing DOM nodes using `Document` methods such as `getElementById`. There are other ways to interact with the DOM which you will learn in time. 

For now, let's consider a more direct way of accessing the value of your `title` element. 

hint: You're already accessing that element. You just need to the value.

Reply:
thank you! I will change it then.